hubspot-client-bundles/hbase-mapreduce-bundle/src/main/resources/hbase-site.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0"?>
+<?xml-stylesheet type="text/xsl" href="configuration.xsl"?>
+<configuration>
+  <property>
+    <name>hbase.io.compress.zstd.codec</name>
+    <value>org.apache.hadoop.hbase.io.compress.zstd.ZstdCodec</value>
```

Comment:
Can you just add a comment ot this file so that we dont have to go searching for why it exists in the future? Something about how Compression.java uses HBaseConfiguration.creatae() so this was the easiest way to ensure all of our clients had this on the classpath, since we can't modify it at runtime

Reply:
Oh I see you added a comment, would appreciate a note about the above gotcha

Reply:
done

Reply:
Thanks! This is a draft so I can't approve it, but LGTM